prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+  const fullName = 'Dylan Dao';
+  console.log(fullName);
+  console.log(typeof fullName);
+  const numberOfPets = 4;
+  console.log(numberOfPets);
+  const likesVegetables = true;
+  console.log(likesVegetables);
+  console.log(typeof likesVegetables);
+  const nothing = null;
+  const never = '';
```

Comment:
this variable should be a `let` variable and it should not be assigned a value, only declared.